@xen-orchestra/lite/src/App/TreeView.tsx

```diff
@@ -41,6 +44,16 @@ const getIconColor = (obj: Host | Vm) => {
 
 const TreeView = withState<State, Props, Effects, Computed, ParentState, ParentEffects>(
   {
+    effects: {
+      setSelectedNodes: function (_, nodeIds) {
+        const { objectId } = this.state
+        this.props.history.push(
+          objectId?.match(/^[0-9a-z]{8}-([0-9a-z]{4}-){3}[0-9a-z]{12}$/) != null
```

Comment:
We need to check the value of `objectId`. Since he depend of the URL, sometime he can be undefined or not an objectId.
ie: `http://.../infrastruture` `http://.../infrastruture/Foo/Bar`